src/test/java/io/debezium/connector/vitess/SourceInfoTest.java

```diff
@@ -97,7 +97,7 @@ public void timestampIsPresent() {
 
     @Test
     public void tableIdIsPresent() {
-        assertThat(source.struct().getString(SourceInfo.DATABASE_NAME_KEY)).isEqualTo("server_foo");
+        assertThat(source.struct().getString(SourceInfo.DATABASE_NAME_KEY)).isEqualTo(TEST_KEYSPACE);
         assertThat(source.struct().getString(SourceInfo.SCHEMA_NAME_KEY)).isEqualTo("s");
```

Comment:
can we assert `SourceInfo.KEYSPACE_NAME_KEY` here as well?

Reply:
Done. Added a separate test `keyspaceIsPresent`.